hocon.gemspec

```diff
@@ -1,13 +1,14 @@
 Gem::Specification.new do |s|
   s.name        = 'hocon'
-  s.version     = '1.1.2'
+  s.version     = '1.2.0'
```

Comment:
FYI we have another way of doing this that, most importantly, allows for a CI job to automatically bump+release the gem.

For example, you'd create a dedicated `version.rb` that looks like:
```ruby
module Hocon
  module Version
    STRING = '1.2.0'
  end
end
```

You'd then have the gemspec reference this variable directly here, and could avoid the call into `Gem.loaded_spec['hocon'].version`.

Existing examples:

- https://github.com/puppetlabs/beaker-hostgenerator/blob/master/lib/beaker-hostgenerator/version.rb
- https://github.com/puppetlabs/beaker/blob/master/lib/beaker/version.rb
- https://github.com/puppetlabs/beaker-pe/blob/master/lib/beaker-pe/version.rb

Not saying we should block the PR on this (or even make the change at all), just wanted to bring it up.